cores/esp8266/lwIPDhcpServer-NonOS.cpp

```diff
@@ -0,0 +1,68 @@
+/*
+ lwIPDhcpServer-NonOS.cpp - DHCP server wrapper
+
+ Copyright (c) 2020 esp8266 arduino. All rights reserved.
+ This file is part of the esp8266 core for Arduino environment.
+
+ This library is free software; you can redistribute it and/or
+ modify it under the terms of the GNU Lesser General Public
+ License as published by the Free Software Foundation; either
+ version 2.1 of the License, or (at your option) any later version.
+
+ This library is distributed in the hope that it will be useful,
+ but WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ Lesser General Public License for more details.
+
+ You should have received a copy of the GNU Lesser General Public
+ License along with this library; if not, write to the Free Software
+ Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301  USA
+ */
+
+// STARTS/STOPS DHCP SERVER ON WIFI AP INTERFACE
+// these functions must exists as-is with "C" interface,
+// nonos-sdk calls them at boot time and later
+
+#include <lwip/init.h> // LWIP_VERSION
+
+#if LWIP_VERSION_MAJOR != 1
```

Comment:
@d-a-v  the code for old lwip can be removed, ref: https://github.com/esp8266/Arduino/pull/7436

BTW: There will be only one LWIP in 3.0? Just asking to be aware of upcoming changes :)

Reply:
Not as easy as code removal. Old lwIP include files are needed to compile the other side of the glue.
I'm moving these files into lwip2 repository.
Yes, lwIP1.4 will not be available in esp8266 Arduino core v3.